### PR TITLE
test: stabilize resouce_metering::test_dynamic_config::test_report_interval

### LINKED ISSUE #12541: unstable resouce_metering::test_dynamic_config::test_report_interval
## Bug Report

https://ci.pingcap.net/blue/organizations/jenkins/tikv_ghpr_test/detail/tikv_ghpr_test/20390/pipeline/114
https://ci.pingcap.net/blue/organizations/jenkins/tikv_ghpr_test/detail/tikv_ghpr_test/20393/pipeline/95

### PR DESCRIPTION
**Pull request by zhongzc**
Signed-off-by: Zhenchi <zhongzc_arch@outlook.com>

Issue Number: close #12541

### What is changed and how it works?

```commit-message
stabilize some integration test
```

### Check List 

Tests 

- Integration test

### Release note 

```release-note
None
```

### CODE REVIEW

**Review by mornyx — APPROVED** [review]

**Review by tonyxuqqi — APPROVED** [review]

File: tests/integrations/resource_metering/test_dynamic_config.rs [hunk 1]
```diff
@@ -76,15 +76,20 @@ pub fn test_report_interval() {
     // | Report Interval |
     // |       1s        |
     test_suite.cfg_report_receiver_interval("1s");
-    let res = test_suite.block_receive_one();
-    let begin = Instant::now();
-    assert!(res.contains_key("req-1"));
```
Comment (iosmanthus) [inline tests/integrations/resource_metering/test_dynamic_config.rs:81]: Why were these two assertions removed?

Comment (zhongzc) [reply tests/integrations/resource_metering/test_dynamic_config.rs:81]: This test doesn't care about this. It was just stricter before.

**Review by iosmanthus — APPROVED** [review]
LGTM

**Review by breezewish — APPROVED** [review]

**Pull request merged**